content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -141,22 +139,6 @@ API connections, and its
 
    ![](/img/gdrive-create-credentials.png)
 
-6. The newly generated **client ID** and **client secret** should be shown to
```

Comment:
@dtrifiro I'm not sure I understand why we want to remove this? Creating a custom APP is still better compared to the built -in one. Also, it's not related to the changes we are doing, or I'm missing something.

Reply:
Looking at this again, this does not make any sense 🤦 Closing this PR as it's not required